src/status_im2/contexts/syncing/setup_syncing/view.cljs

```diff
@@ -38,30 +38,30 @@
 
 (defn view
   []
-  (let [profile-color (:color (rf/sub [:onboarding-2/profile]))
-        valid-for-ms  (reagent/atom code-valid-for-ms)
-        code          (reagent/atom nil)
-        delay         (reagent/atom nil)
-        timestamp     (reagent/atom nil)
-        set-code      (fn [connection-string]
-                        (when (sync-utils/valid-connection-string? connection-string)
-                          (reset! timestamp (* 1000 (js/Math.ceil (/ (datetime/timestamp) 1000))))
-                          (reset! delay 1000)
-                          (reset! code connection-string)))
-        clock         (fn []
-                        (if (pos? (- code-valid-for-ms
-                                     (- (* 1000 (js/Math.ceil (/ (datetime/timestamp) 1000)))
-                                        @timestamp)))
-                          (swap! valid-for-ms (fn [_]
-                                                (- code-valid-for-ms
-                                                   (- (* 1000
-                                                         (js/Math.ceil (/ (datetime/timestamp) 1000)))
-                                                      @timestamp))))
-                          (reset! delay nil)))
-        cleanup-clock (fn []
-                        (reset! code nil)
-                        (reset! timestamp nil)
-                        (reset! valid-for-ms code-valid-for-ms))]
+  (let [valid-for-ms                           (reagent/atom code-valid-for-ms)
+        {:keys [customization-color]}          (rf/sub [:profile/multiaccount])
```

Comment:
this sub looks wrong as syncing is both in app and a login method.
I recently created a `sub` which should work for these pages
you can use 
`customization-color (rf/sub [:profile/customization-color])`